project/Build.scala

```diff
@@ -1,14 +1,14 @@
 import sbt._
 import Keys._
-import PlayProject._
+import play.Project._
 
 object ApplicationBuild extends Build {
 
     val appName         = "collins"
-    val appVersion      = "1.2.4-SNAPSHOT"
```

Comment:
I can handle cutting a new release, announcements and the like, once this is merged.